deploy/swrs/parameters/table/fuel_charge_001.sql

```diff
@@ -0,0 +1,78 @@
+-- Deploy ggircs:swrs/parameters/table/fuel_charge_001 to pg
+-- requires: swrs/parameters/table/fuel_charge
+
+begin;
+
+
+update ggircs_parameters.fuel_charge set end_date='2023-03-31' where start_date='2022-04-01' and carbon_tax_act_fuel_type_id=1;
+insert into ggircs_parameters.fuel_charge (fuel_charge, start_date, end_date, carbon_tax_act_fuel_type_id, metadata) values (0.1592, '2023-04-01', '9999-12-31', 1, null);
+
+update ggircs_parameters.fuel_charge set end_date='2023-03-31' where start_date='2022-04-01' and carbon_tax_act_fuel_type_id=2;
+insert into ggircs_parameters.fuel_charge (fuel_charge, start_date, end_date, carbon_tax_act_fuel_type_id, metadata) values (0.1431, '2023-04-01', '9999-12-31', 2, null);
+
+update ggircs_parameters.fuel_charge set end_date='2023-03-31' where start_date='2022-04-01' and carbon_tax_act_fuel_type_id=3;
+insert into ggircs_parameters.fuel_charge (fuel_charge, start_date, end_date, carbon_tax_act_fuel_type_id, metadata) values (0.1593, '2023-04-01', '9999-12-31', 3, null);
```

Comment:
Looks like a copy paste error? 0.1593 is the 2022 rate. Should be 0.2072
```suggestion
insert into ggircs_parameters.fuel_charge (fuel_charge, start_date, end_date, carbon_tax_act_fuel_type_id, metadata) values (0.2072, '2023-04-01', '9999-12-31', 3, null);
```